docs/rule/require-form-method.md

```diff
@@ -0,0 +1,40 @@
+# require-form-method
+
+This rule requires all `<form>` elements to have `method` attribute with `POST` or `GET` value.
+
+By default `form` elements without `method` attribute are submitted as `GET` requests.
+In usual applications `submit` event listeners are attached to `form` elements and `event.preventDefault()` is called to avoid form submission.
+
+However in case of failure to prevent default action, form submission as `GET` request can leak sensitive end-user information.
+Examples of valid uses for `GET` requests are non-secure data, bookmarking the submission result, data search query strings.
+
+**Caution** - this rules does not check for `formmethod` attribute on `form` elements themselves.
+
+## Examples
+
+This rule **forbids** the following:
+
+```hbs
+<form>Hello world!</form>
+<form method="">Hello world!</form>
+<form method="random">Hello world!</form>
+```
+
+This rule **allows** the following:
+
+```hbs
+<form method="post">Hello world!</form>
+<form method="get">Hello world!</form>
+```
+
+## Configuration
+
+The following values are valid configuration:
+
+* boolean - `true` to enable / `false` to disable
+* object -- An object with the following keys:
+  * `allowedMethods` -- An array of allowed form `method` attribute values, default: `['POST', 'GET']`
+
+## References
```

Comment:
This is a good, simple link too: https://www.w3schools.com/tags/att_form_method.asp

Reply:
Added.